src/main/webapp/WEB-INF/static/angular-templates/showSettingFormElement.html

```diff
@@ -1,31 +1,19 @@
 <div ng-switch="widgetType">
     <div ng-switch-when="DROPDOWN">
-        <ui-select name="name" theme="select2" search-enabled="false" ng-model="valuecontainer[targetkey]" ng-disabled="blockInput" style="width:100%" ng-change="changefunction()"
-                   append-to-body="true">
-            <ui-select-match placeholder="Please Choose">{{valuecontainer[targetkey].description}}</ui-select-match>
-            <ui-select-choices repeat="option in dropdownValues">
-                <div ng-bind-html="option.description"></div>
-            </ui-select-choices>
-        </ui-select>
+        <div ng-if="!isLocation && !isBreedingMethod">
+            <ui-select name="name" theme="select2" search-enabled="false" ng-model="valuecontainer[targetkey]" ng-disabled="blockInput" style="width:100%" ng-change="changefunction()"
+                       append-to-body="true">
+                <ui-select-match placeholder="Please Choose">{{valuecontainer[targetkey].description}}</ui-select-match>
+                <ui-select-choices repeat="option in dropdownValues">
+                    <div ng-bind-html="option.description"></div>
+                </ui-select-choices>
+            </ui-select>
+        </div>
         <div ng-if="isLocation">
-            <div class="possibleValuesDiv">
-                <input type="radio" ng-disabled="blockInput"
-                 ng-model="lookupLocation" value=1 ng-change="updateDropdownValuesBreedingLocation()"> &nbsp;
-                <span th:text="#{show.breeding.location}">Breeding locations</span> &nbsp;
-                <input type="radio" ng-disabled="blockInput" 
-                ng-model="lookupLocation" value=2 ng-change="updateDropdownValuesAllLocation()"> &nbsp;
-                <span th:text="#{show.all.location}">All locations types</span>
-            </div>
-            <div class="possibleValuesDiv">            
-                <input type="checkbox" ng-model="localData.useFavorites" ng-change="updateDropdownValuesFavorites()"> &nbsp;
-                <span th:text="#{show.favorite.location}">Show only favorite locations </span> &nbsp;
-	            <span>
-	                <a href="javascript: void(0)" th:text="#{study.managesettings.manage.location}" ng-click="blockInput || initiateManageLocationModal()">Manage Location</a>
-	            </span>
-            </div>
+            <locations-select valuecontainer="valuecontainer" targetkey="targetkey" on-location-select="changefunction()"></locations-select>
```

Comment:
I believe we don't add Location variable in both Trial Settings and Treatment Factors. I think this can be removed.